schema.graphql

```diff
@@ -3,6 +3,7 @@ type Trader @entity {
   totalTradeAmount: BigInt! #calculated the Trade event increase or decrease based on tokenGive and tokenGet
```

Comment:
```suggestion
  totalTradeAmount: BigInt! # toal selled amount of KSQT
```

Reply:
As we use `totalTradeAmount` to calculate the `maxTradeAmount`, so that we only care about the sold KSQT amount